tests/CommunityToolkit.Aspire.Hosting.Minio.Tests/AppHostTests.cs

```diff
@@ -0,0 +1,43 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using Aspire.Components.Common.Tests;
+using Aspire.Hosting;
+using CommunityToolkit.Aspire.Testing;
+using Microsoft.Extensions.Configuration;
+using Microsoft.Extensions.Hosting;
+
+namespace CommunityToolkit.Aspire.Hosting.Minio.Tests;
+
+[RequiresDocker]
+public class AppHostTests(AspireIntegrationTestFixture<Projects.CommunityToolkit_Aspire_Hosting_Minio_AppHost> fixture) : IClassFixture<AspireIntegrationTestFixture<Projects.CommunityToolkit_Aspire_Hosting_Minio_AppHost>>
+{
+    [Fact]
+    public async Task ResourceStartsAndRespondsOk()
+    {
+        var resourceName = "apiservice";
+        await fixture.ResourceNotificationService.WaitForResourceHealthyAsync(resourceName).WaitAsync(TimeSpan.FromMinutes(5));
+        var httpClient = fixture.CreateHttpClient(resourceName);
+
+        var response = await httpClient.GetAsync("/");
+
+        Assert.Equal(HttpStatusCode.OK, response.StatusCode);
+    }
+    
+    [Fact]
+    public async Task ApiServiceCreateData()
```

Comment:
This test has three separate waiting timeouts, do we really want to wait a potential 15 minutes for the timeout if something is wrong? Could we not instead have a cancellation token that is shared between each call, and maybe reduce the duration?